osmaxx-py/config/settings/common.py

```diff
@@ -50,7 +50,12 @@
 LOCAL_APPS = (
     'osmaxx.excerptexport',
     'osmaxx.social_auth',
+    'excerptconverter.dummyexcerptconverter'
```

Comment:
You mean: Rename excerptconverter/dummyexcerptconverter to excerptconverter/dummy?